enhancements/console/quick-starts.md

```diff
@@ -0,0 +1,154 @@
+      review: 
+        instructions: Make sure the Serverless Operator was successfully installed ...
+        taskHelp: Try walking through the steps again to properly install the Serverless Operator
+      recapitulation:
+        success: You've just installed the Serverless Operator! Next, we'll install the required CR's for this Operator to run.
+        failed: Check your work to make sure that the Serverless Operator is properly installed
+    - title: Create knative-serving API
+      description: The first CR we'll create is knative-serving ...
+      review:
+        instructions: Make sure the knative-serving API was successfully installed ...
+        taskHelp: Try walking through the steps again to properly create the instance of knative-serving
+      recapitulation:
+        success: You've just created an instance of knative-serving! Next, we'll create an instance of knative-eventing
+        failed: Check your work to make sure that the instance of knative-serving is properly created
+    - title: create knative-eventing API
+      description: The second CR we'll create is knative-eventing ...
+      review:
+        instructions: Make sure the knative-eventing API was successfully installed ...
+        taskHelp: Try walking through the steps again to properly create the instance of knative-eventing
+      recapitulation:
+        success: You've just created an instance of knative-eventing!
+        failed: Check your work to make sure that the instance of knative-eventing is properly created
+  conclusion: Your Serverless Operator is ready! If you want to learn how to deploy a serverless application, take the Serverless Application tour.
+  nextQuickStart: serverless-application
+```
+
+#### Quick Start icon
+
+Each Quick Start CR can contain an icon that is specified in the `spec.icon` field. This field is either an URL of the Package Manifest that contains icon, or a base64 encoded image that are used as Quick Start's icon
+
+#### Quick Start's Access Review
+
+Since different users might not have access to specific actions on different resources, it would be pointless for them to go though a Quick Start and be unable to complete its tasks. For that reason the `spec` of the Quick Start CRD contains also `accessReviewResources` field, that should contain an array of resources and actions done on them, that user should perform during taking the Quick Start tour. User's access to the Quick Start is reviewed
+based on array of resource actions. In order for the user to see the Quick Start and take its tour, his access review needs to pass all the listed resources actions.
+
+#### Air Gapped Environments
+
+Since the supported Quick Starts CRs will be part of the `console-operator`'s manifests, they will be distributed together with image.
+
+### Third party Quick Starts
+
+Third party Quick Starts will need to be created manually on the cluster, or upon operator installation.
```

Comment:
I'd make it clear that it's the operator itself that creates these after installation.

Reply:
How about?

```
### Installation

`console-operator` is responsible for installing supported Quick Starts at the end of the cluster provisioning process. Third party Quick Starts will need to be created manually on the cluster.
```

Reply:
I was thinking something like

```suggestion
Third party Quick Starts will need to be created by an operator after it's installed or manually by a cluster administrator.
```